### PR TITLE
Add some tests to increase coverage

### PR DESCRIPTION
**Pull request by Byczong**

### CODE REVIEW

File: api/tests/test_nameguard.py [hunk 1]
```diff
@@ -332,3 +332,24 @@ async def test_empty_labels(nameguard: NameGuard):
     assert r.labels[2].rating is Rating.PASS
     assert r.labels[2].highest_risk is None
     assert r.labels[2].normalization is Normalization.NORMALIZED
+
+
+
+
+def test_generic_check_result_operators():
+    assert GenericCheckResult(**{'check': Check.NORMALIZED, 'status': CheckStatus.INFO, '_name_message': ''}) == \
```
Comment (Carbon225) [inline api/tests/test_nameguard.py:340]: Why the dictionary?

**Review by Carbon225 — CHANGES_REQUESTED** [review]
```python
GenericCheckResult(**{'check': Check.NORMALIZED, 'status': CheckStatus.INFO, '_name_message': ''})
```
?

File: api/tests/test_nameguard.py (hunk 1 above)
Comment (Byczong) [reply api/tests/test_nameguard.py]: I changed it to keywords

File: api/tests/test_checks.py [hunk 2]
```diff
@@ -14,6 +14,15 @@ def analyse_grapheme(nameguard: NameGuard, grapheme: str):
     return nameguard.analyse_label(grapheme).graphemes[0]
 
 
+def test_ordering():
+    assert CheckStatus.SKIP < CheckStatus.INFO < CheckStatus.PASS < CheckStatus.WARN < CheckStatus.ALERT
+    assert CheckStatus.WARN > CheckStatus.INFO
+    assert CheckStatus.SKIP == CheckStatus.SKIP
+    assert CheckStatus.ALERT != CheckStatus.WARN
+    assert CheckStatus.SKIP <= CheckStatus.INFO
+    assert CheckStatus.WARN >= CheckStatus.WARN
```
Comment (djstrong) [inline api/tests/test_checks.py:23]: is it ok?

Comment (Carbon225) [reply api/tests/test_checks.py:23]: Yes

File: api/tests/test_nameguard.py [hunk 3]
```diff
@@ -332,3 +332,23 @@ async def test_empty_labels(nameguard: NameGuard):
...
+    assert GenericCheckResult(check=Check.NORMALIZED, status=CheckStatus.WARN, _name_message='') <= \
+           GenericCheckResult(check=Check.FONT_SUPPORT, status=CheckStatus.ALERT, _name_message='')
+    assert GenericCheckResult(check=Check.NORMALIZED, status=CheckStatus.PASS, _name_message='') > \
+           GenericCheckResult(check=Check.CONFUSABLES, status=CheckStatus.PASS, _name_message='')
+    assert GenericCheckResult(check=Check.NORMALIZED, status=CheckStatus.ALERT, _name_message='') >= \
+           GenericCheckResult(check=Check.FONT_SUPPORT, status=CheckStatus.ALERT, _name_message='')
+
+
+def test_generic_check_result_repr():
+    assert repr(GenericCheckResult(**{'check': Check.NORMALIZED, 'status': CheckStatus.PASS, '_name_message': ''})) == \
```
Comment (Carbon225) [inline api/tests/test_nameguard.py:353]:
```suggestion
    assert repr(GenericCheckResult(check=Check.NORMALIZED, status=CheckStatus.PASS, _name_message='')) == \
```

**Review by Carbon225 — CHANGES_REQUESTED** [review]

**Review by Carbon225 — APPROVED** [review]

**Review by notrab — APPROVED** [review]

**Pull request merged**